content/sensu-go/5.15/guides/use-federation.md

```diff
@@ -0,0 +1,167 @@
+
+## Step 1 Configure clusters with shared JWT keys
+
+JWTs support asymmetric encryption using RSA or Elliptic Curve keys. Use the `openssl` command line tool to generate a P-256 Elliptic Curve key:
+
+```shell
+openssl ecparam -genkey -name prime256v1 -noout -out ec_private.pem
+```
+
+and then generate a public key from the private key:
+
+```shell
+openssl ec -in ec_private.pem -pubout -out ec_public.pem
+```
+
+After generating these keys, copy them to your Sensu clusters. For this guide, we'll put JWT keys into `/etc/sensu/certs` and use the [`jwt-private-key-file` and `jwt-public-key-file` attributes][4] in `/etc/sensu/backend.yml` to specify the paths to these JWT keys:
+
+{{< highlight yml >}}
+jwt-private-key-file: /etc/sensu/certs/ec_private.pem
+jwt-public-key-file: /etc/sensu/certs/ec_public.pem
+{{< /highlight >}}
+
+After updating the backend configuration, restart `sensu-backend` for those settings to take effect.
+
+## Step 2 Create etcd replicators
+
+You can use etcd Replicators to synchronize [RBAC policy resources][10] between clusters. This allows you to centrally define permissions that replicate to all federated clusters, ensuring consistent access for Sensu users.
+
+Etcd replicators use the [etcd make-mirror utility][12] for one-way key replication. Our [etcd-replicators reference][2] includes [examples][9] for `Role`, `RoleBinding`, `ClusterRole`, and `ClusterRoleBinding` resources.
+
+## Step 3 Register clusters
+
+Each registered cluster must have a name and a list of cluster member URLs corresponding to the backend REST API.
+
+### Register a single cluster
+
+You do not need to register the cluster that you are currently operating from (for example, self or local-cluster) unless you want to configure the cluster name.
+
+From `local-cluster`, run `sensuctl create` on the yaml or JSON below to register cluster `us-west-2a`:
+
+{{< language-toggle >}}
+
+{{< highlight yml >}}
+api_version: federation/v1
+type: Cluster
+metadata:
+  name: us-west-2a
+spec:
+  api_urls:
+  - http://10.0.0.1:8080
+  - http://10.0.0.2:8080
+  - http://10.0.0.3:8080
+{{< /highlight >}}
+
+{{< highlight json >}}
+{
+  "api_version": "federation/v1",
+  "type": "Cluster",
+  "metadata": {
+    "name": "us-west-2a"
+  },
+  "spec": {
+    "api-urls": [
+      "http://10.0.0.1:8080",
+      "http://10.0.0.2:8080",
+      "http://10.0.0.3:8080"
+    ]
+  }
+}
+{{< /highlight >}}
+
+{{< /language-toggle >}}
+
+### Register additional clusters
+
+From `local-cluster`, run `sensuctl create` on the yaml or JSON below to register an additional cluster (with the same API URLs as `local-cluster`) and define the name as `us-west-2b`:
```

Comment:
Maybe we should note that this will not overwrite `local-cluster`, rather it will only create a duplicate with a custom name.